student-cvs/tefis96-9675/index.html

```diff
@@ -0,0 +1,206 @@
+                    <a href="https://github.com/StephanyCS1/Calculadora"><img class="icon"
+                            src="https://img.icons8.com/nolan/512/github.png" alt="github"></a>
+                </div>
+            </div>
+            <div class="contProjects">
+                <div class="img">
+                    <img class="imgProjec" src="img/pk-re.png" alt="">
+                </div>
+                <div class="tech">
+                    <img class="iconTech" src="https://icongr.am/devicon/html5-original.svg?size=34&color=currentColor"
+                        alt="HTML">
+                    <img class="iconTech" src="https://lenguajecss.com/assets/logo.svg" alt="CSS">
+                    <img class="iconTech"
+                        src="https://icongr.am/devicon/javascript-original.svg?size=34&color=currentColor"
+                        alt="JavaScript">
+                    <img class="iconTech" src="https://www.datocms-assets.com/45470/1631110818-logo-react-js.png"
+                        alt="React">
+                </div>
+                <div class="description">
+                    <h3>Pokedex React</h3>
+                    <p>Busqueda según tipo de pokemon, o nombre o iniciales, detalle de estadisticas por pokemón</p>
+                </div>
+                <div class="repo">
+                    <a id="repo" href="https://github.com/StephanyCS1/Pokedex-React"><img class="icon"
+                            src="https://img.icons8.com/nolan/512/github.png" alt="github"></a>
+                </div>
+            </div>
+        </div>
+    </section>
+    <section class="contact" id="contact">
+        <h2>Contacto</h2>
+        <div class="contactInfo">
+            <a href="https://www.linkedin.com/in/stephany-castro-salas-03a001172/"><img class="iconContact"
+                    src="https://icongr.am/devicon/linkedin-original.svg?size=34&color=d4b5b5" alt="linkedin"></a>
+            <a href="https://github.com/StephanyCS1"><img class="iconContact" src="https://img.icons8.com/nolan/512/github.png"
+                    alt="github"></a>
+        </div>
+
+        <div class="contactForm">
+            <form action="/server" method="post">
+                <label for="name">Nombre</label>
+                <input type="text" name="nameField" id="name" placeholder="Escriba su nombre" required>
+                <label for="email">Correo Electronico</label>
+                <input type="email" name="emailField" id="email" placeholder="Escriba su correo electronico" required>
+                <label for="describe" class="describe">
+                    Mensaje
+                    <textarea name="describeField" id="describe" cols="30" rows="10"
+                        placeholder="Porfavor escriba su mensaje" required></textarea>
+                </label>
+                <div class="button">
+                    <a href="mailto:stephanycastrosalas@gmail.com"><button type="submit">Enviar</button></a>
```

Comment:
Hola, al pasarlo por el validador de html semántico, me salió esto:
Error: The element button must not appear as a descendant of the a element.
Quizás solo debas usar el elemento button y no sea necesario usar el elemento <a>.

Reply:
Hola, ya corregí el error, no lo había visto
Muchas gracias 😁